src/scripts/modules/media/body/content-baked.less

```diff
@@ -106,6 +106,15 @@ iframe {
   }
 }
 
+figure.splash.unnumbered {
+  display: table;
```

Comment:
Can this change be made using a `block` or `inline block` display or by using flexbox with a column flow? Typically `display: table` is no longer required for most tasks.

Reply:
@tomjw64 That's a copy-paste from the styling for `os-figure` (see line 122 here https://github.com/openstax/webview/blob/0a1d53c49152eb82da6d6d6788886275b687c96d/src/scripts/modules/media/body/content-baked.less ). I thought that it was set like this for a reason which I can't test and didn't want to do the very same styling in a different way. 
Should I update the styling for both `figure.splash.unnumbered` and `os-figure`?

Reply:
Fair point. Approved, in that case.